src/routes/resource-edit.js

```diff
@@ -66,18 +66,6 @@ class ResourceEditRoute extends Component {
 
     if (values.isSelected === false && model.package.isCustom) {
       destroyResource(model);
-    } else if (values.isSelected === false) {
```

Comment:
🤔 This one doesn't seem right. Do we have a test covering the deselection (and submitting) the resource on the edit screen?

Reply:
Nope, you're right, we need this. We don't have anything currently testing that. I'm setting that up now and putting this chunk back in. Complete oversight on my part haha.